controllers/cloud.redhat.com/clowdjobinvocation_controller.go

```diff
@@ -49,6 +57,10 @@ type ClowdJobInvocationReconciler struct {
 	Recorder record.EventRecorder
 }
 
+var IqeClowdJob = providers.NewSingleResourceIdent("iqeclowdjob", "core_iqe_clowdjob", &batchv1.Job{})
```

Comment:
The first argument to this is supposed to be a provider string, so I think setting it to `cji` in all three cases is most appropriate